tests/Functional/Filter/SuffixFilterTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Functional\Filter;
+
+use Paraunit\Filter\Filter;
+use Tests\BaseFunctionalTestCase;
+
+class SuffixFilterTest extends BaseFunctionalTestCase
+{
+    protected function setup(): void
+    {
+        $this->setOption('configuration', $this->getStubPath() . DIRECTORY_SEPARATOR . 'phpunit_with_2_testsuites.xml');
+        $this->setOption('test-suffix', 'GreenTestStub.php');
+
+        parent::setup();
+    }
```

Comment:
Same here: we need to test the comma-separated case

Reply:
Added test for comma separated value